### PR TITLE
Honor visible devices

### PR DESCRIPTION
**Pull request by Delaunay**

### CODE REVIEW

**Comment by Delaunay** [comment]
The GPU index 0,1,2,3,4 gets replaced by a much uglier one `GPU-e6601185-4fcd-d20f-055f-89b243800913` to support MIG GPUs. I think it might make the report unreadable inside milabench we might want to do some remapping to get back indexes if possible.

**Comment by Delaunay** [comment]
So I used the index back, as we do not need to support MIG yet.
So now we can filter by UUID and index, the output is unchanged, it might work with MIG too if MIG has valid Indexes but I am not sure if it is the case

**Comment by Delaunay** [comment]
@breuleux, it is ready for review, finished testing it on milabench, works well for both per_gpu and multi-gpu configs.

File: voir/instruments/gpu/cpu.py [hunk 1]
```diff
@@ -3,7 +3,7 @@ def is_installed():
 
 
 class DeviceSMI:
-    def get_gpus_info(self):
+    def get_gpus_info(self, selection=ModuleNotFoundError):
```
Comment (breuleux) [inline voir/instruments/gpu/cpu.py]: What is `ModuleNotFoundError` doing here?

**Review by breuleux — APPROVED** [review]

File: voir/instruments/gpu/cpu.py (hunk 1 above)
Comment (Delaunay) [reply voir/instruments/gpu/cpu.py]: probably some copy paste error, fixed

**Pull request merged**